.pre-commit-hooks.yaml

```diff
@@ -1,21 +1,21 @@
 -   id: fmt
     name: fmt
-    description: Format files with rustfmt.
-    entry: rustfmt --
+    description: Format files with cargo fmt.
+    entry: cargo fmt --
```

Comment:
Is the couble-dash actually needed ? I think is will make it hard to pass args to the command should someone wisth to do it. Ref #7

Reply:
Anything that goes after the double-dash is actually then sent to rustfmt. But you're right, adding the double-dash prevents the user to sending arguments to cargo fmt. I'll remove it.

Reply:
Guess I should have tested it first, turns out removing the double dash actually breaks the hook. They are needed for pre-commit to be able to pass file paths.